packages/react-reconciler/src/__tests__/ReactSuspense-test.internal.js

```diff
@@ -389,17 +389,6 @@ describe('ReactSuspense', () => {
     expect(root).toMatchRenderedOutput('Hi');
   });
 
-  it('throws if tree suspends and none of the Suspense ancestors have a boundary', () => {
```

Comment:
Deleted this test because it was a duplicate of the one in ReactSuspenseWithNoopRenderer